causal_testing/testing/causal_test_suite.py

```diff
@@ -13,15 +13,15 @@ class CausalTestSuite(UserDict):
     base_test_case's and execute each causal_test_case with each iterator.
     """
 
-    def add_test_object(self, base_test_case, causal_test_case_list, estimators, estimate_type: str = "ate"):
+    def add_test_object(self, base_test_case, causal_test_case_list, estimators_classes, estimate_type: str = "ate"):
```

Comment:
Should `estimators_classes` have a type in the header? I guess it's type would be `type`. This all ties in with the ABC stuff which I get a little lost in sometimes.

Reply:
`Iterable[Type[Estimator]]`

I've chosen the above as the type hint as it should be a list or any iterable of casses that inherit from the `Estimator` class. 

That type hint helps me understand the datatype, but it's hard to say if there is a more technically accurate type hint to do with typing ABCs?

Reply:
Not that I'm aware of.